### PR TITLE
Add 7.11 release notes

### PR DESCRIPTION
**Pull request by pugnascotia**
First draft of the release notes and highlights for 7.11.0

### CODE REVIEW

**Review by jrodewig — COMMENTED** [review]
Thanks for putting this together @pugnascotia. I left some suggestions and asked for a few other teams to take a look. After they're done, I'll take another look.

File: docs/reference/release-notes/highlights.asciidoc [hunk 1]
```diff
@@ -23,27 +24,83 @@ Other versions:
 | {ref-bare}/7.1/release-highlights-7.1.0.html[7.1]
 | {ref-bare}/7.0/release-highlights-7.0.0.html[7.0]
 
+// tag::notable-highlights[]
+[discrete]
+=== Add support for runtime fields
```
Comment (jrodewig) [inline docs/reference/release-notes/highlights.asciidoc:29]:
```suggestion
=== Runtime fields
```

File: docs/reference/release-notes/highlights.asciidoc [hunk 2]
```diff
@@ -23,27 +24,83 @@ Other versions:
...
+{"type":"audit", "timestamp":"2020-09-26T12:58:52,955+0300", "node.id":"6BXu_9j6QPK7jvmOwzxqLQ", "event.type":"security_config_change", "event.action":"put", "request.id":"i2XtJLCoRheGuwUdCXjDJw", "config_change":{"put_role":{"name":"role_fls","cluster_privileges":["all"],"run_as":[],"indices_privileges":[{"names":["apm*"],"privileges":["read"],"field_security":{"grant":["granted"]},"query":"{\"term\": {\"service.name\": \"bar\"}}","allow_restricted_indices":false},{"names":["apm-all*"],"privileges":["all"],"query":"{\"term\": {\"service.name\": \"bar2\"}}","allow_restricted_indices":false}],"application_privileges":[],"metadata":{},"configurable_cluster_privileges":{}}}}
+----
+// NOTCONSOLE
+
+[discrete]
+=== EQL: Wildcard and list lookup support for the `:` operator
+
+EQL's case-insensitive `:` operator now supports the `*` and `?` wildcards. The
+`*` wildcard matches zero or more characters. The `?` wildcard matches exactly
+one character. You can now also use `:` as a lookup operator to compare a string
```
Comment (jrodewig) [inline docs/reference/release-notes/highlights.asciidoc:91]: I introduced some awk wording here. Sorry.
```suggestion
one character. You can also now use `:` as a lookup operator to compare a string
```

**Review by jrodewig — APPROVED** [review]
LGTM. I've added the security and ML docs teams as reviewers for their sections.

Thanks @pugnascotia.

**Review by tvernum — APPROVED** [review]

**Comment by pugnascotia** [comment]
@elastic/ml-docs could someone review the 7.11 notes, please?

File: docs/reference/release-notes/7.11.asciidoc [hunk 3]
```diff
@@ -0,0 +1,317 @@
...
+* Adding a warning header when a license is about to expire {es-pull}64948[#64948] (issue: {es-issue}60562[#60562])
+
+Machine Learning::
+* Add new setting `xpack.ml.use_auto_machine_memory_percent` for auto calculating native memory percentage allowed for jobs {es-pull}63887[#63887] (issue: {es-issue}63795[#63795])
+* Add new snapshot upgrader API for upgrading older snapshots {es-pull}64665[#64665] (issue: {es-issue}64154[#64154])
+* Add search `runtime_mappings` to datafeed configuration {es-pull}65606[#65606]
+* Add total ML memory to ML info {es-pull}65195[#65195] (issue: {es-issue}64225[#64225])
+* Adding `result_type` and `mlcategory` fields to category definitions {es-pull}63326[#63326] (issue: {es-issue}60108[#60108])
+* Increase log level for forecast disk storage problems {es-pull}64766[#64766] (issue: {es-issue}58806[#58806])
+* Provide a way to revert an AD job to an empty snapshot {es-pull}65431[#65431]
```
Comment (szabosteve) [inline docs/reference/release-notes/7.11.asciidoc:188]:
```suggestion
* Provide a way to revert an {anomaly-job} to an empty snapshot {es-pull}65431[#65431]
```

**Review by szabosteve — APPROVED** [review]
One minor comment on ML docs, otherwise LGTM.

**Pull request merged**